src/scripts/main.js

```diff
@@ -1,5 +1,25 @@
 'use strict';
 
 document.addEventListener('click', (e) => {
-  // write code here
+  const spider = document.querySelector('.spider');
+  const wall = document.querySelector('.wall');
+
+  if (e.target !== wall) {
+    return;
+  }
+
+  const posX = e.clientX;
+  const posY = e.clientY;
+
+  const wallRect = wall.getBoundingClientRect();
+  const spiderRect = spider.getBoundingClientRect();
+
+  let x = posX - wallRect.left - spiderRect.width / 2 - 10;
+  let y = posY - wallRect.top - spiderRect.height / 2 - 10;
```

Comment:
The subtraction of 10 from both `x` and `y` seems arbitrary. Ensure that this offset is intentional and correctly aligns with your design requirements. If it's meant to account for padding or margins, consider defining it as a constant or variable for clarity and maintainability.